.sdkmanrc

```diff
@@ -0,0 +1,6 @@
+# Enable auto-env through the `sdkman_auto_env` config.
+# Add `<key>=<value>` pairs of SDKs to use below.
+#
+# @see https://sdkman.io/usage/#env-command
+
+java=22.0.2-oracle
```

Comment:
What the purpose of this file? Is it required by Capacitor.js? If not, remove it.

Reply:
The purpose is to set recommended Java SDK version for APK building

Reply:
Point to Capacitor docs

Reply:
Capacitor docs not mentioning SDKMan or NVM as well

Reply:
It is not about Capacitor, it is about version management

Reply:
As we found out, Capacitor is sensitive to Java SDK version, setting inappropriate might cause unpredictable errors on building or installation, so I think it is good practice to declare recommended version for reproducible environment across developers

Reply:
Android Studio already comes with a bundled JDK, so the project should build successfully using the version included by default. SDKMan is intended for managing multiple Java versions. We don't need this, and I also doubt that anyone else on the team is using this file.

Reply:
From Cacapitor docs:

> Note: You do not need to separately install the Java Development Kit (JDK). Android Studio will automatically install the proper JDK for you.

https://capacitorjs.com/docs/getting-started/environment-setup#android-requirements